src/components/v2/devtronStackManager/DevtronStackManager.component.tsx

```diff
@@ -429,91 +429,214 @@ export const InstallationWrapper = ({
     upgradeVersion,
     isUpgradeView,
     isActionTriggered,
+    releaseNotes,
     updateActionTrigger,
+    showPreRequisiteConfirmationModal,
+    setShowPreRequisiteConfirmationModal,
+    preRequisiteChecked,
+    setPreRequisiteChecked,
 }: InstallationWrapperType): JSX.Element => {
     const history: RouteComponentProps['history'] = useHistory()
     const location: RouteComponentProps['location'] = useLocation()
     const latestVersionAvailable = isLatestVersionAvailable(serverInfo?.currentVersion, upgradeVersion)
+    const [preRequisiteList, setPreRequisiteList] = useState<
+        { version: string; prerequisiteMessage: string; tagLink: string }[]
+    >([])
+    useEffect(() => {
+        if (releaseNotes) {
+            const _preRequisiteList = []
+            for (let index = 0; index < releaseNotes.length; index++) {
+                const element = releaseNotes[index]
+                if (serverInfo && serverInfo.currentVersion === element.releaseName) {
```

Comment:
Can do this instead `serverInfo?.currentVersion`

Reply:
done